modules/apps/transfer/keeper/msg_server_test.go

```diff
@@ -125,3 +126,48 @@ func (suite *KeeperTestSuite) TestMsgTransfer() {
 		})
 	}
 }
+
+// TestUpdateParams tests UpdateParams rpc handler
+func (suite *KeeperTestSuite) TestUpdateParams() {
+	var emptyAddr string
```

Comment:
Is there any need for this var? Can we just use a string literal below in the testcase?

Reply:
done